datasets/bsd500_cbsd68.py

```diff
@@ -77,8 +77,9 @@ def get_data(self):
             transforms.ToTensor()
         ])
 
-        train_dataset = ImageDataset(
-            config.get_data_path("BSD500") / "train", transform=transform
+        path = get_data_path("BSD500")
+        train_dataset = dinv.datasets.BSDS500(
+            path, download=True, splits='train', transform=transform
```

Comment:
To get all 500 images, you need:
```suggestion
            path, download=True, splits=["train", "val", "test"], transform=transform
```

Reply:
Only the train images are used in this case

Reply:
But I think it is a mistake in the rational of these datasets.
The idea is to say: train/tune params on bsd500, and then test on another image dataset 

So by default, I would use all 500 images instead of only 200.

Reply:
Alright